source/Cosmos.System2/Graphics/Canvas.cs

```diff
@@ -301,50 +299,50 @@ public virtual void DrawLine(Pen pen, int x1, int y1, int x2, int y2)
 
             if (dy == 0) /* The line is horizontal */
             {
-                DrawHorizontalLine(pen, dx, x1, y1);
+                DrawHorizontalLine(color, dx, x1, y1);
                 return;
             }
 
             if (dx == 0) /* the line is vertical */
             {
-                DrawVerticalLine(pen, dy, x1, y1);
+                DrawVerticalLine(color, dy, x1, y1);
                 return;
             }
 
             /* the line is neither horizontal neither vertical, is diagonal then! */
-            DrawDiagonalLine(pen, dx, dy, x1, y1);
+            DrawDiagonalLine(color, dx, dy, x1, y1);
         }
 
         /// <summary>
         /// Draw line.
         /// </summary>
-        /// <param name="pen">Pen to draw with.</param>
+        /// <param name="color">Color to draw with.</param>
         /// <param name="p1">Staring point.</param>
         /// <param name="p2">End point.</param>
         /// <exception cref="ArgumentOutOfRangeException">
         /// <list type="bullet">
-        /// <item>Thrown if pen is null.</item>
+        /// <item>Thrown if color is null.</item>
```

Comment:
To be removed alongside the null-check

**NOTE:** There are a lot of instances in the documentation where this exception is documented to be thrown, but with the removal of the null-check, these would no longer apply. I don't want to duplicate the same comment over and over again, so I'm only going to leave this one, but there's more lines to update